recipes/svtjpegxs/all/patches/0.9.0-0001-manage-fpic.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e7cdc8d..6a88a89 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -88,7 +88,6 @@ else()
+     set_property(CACHE SANITIZER PROPERTY STRINGS Address Memory Thread Undefined Integer)
+ endif()
+ 
+-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
This is set twice in the CMakeLists, maybe they really do need this to properly compile as shared?